storage/container_storage_interface/persistent-storage-csi-azure.adoc

```diff
@@ -0,0 +1,36 @@
+[id="persistent-storage-csi-azure"]
+= Azure CSI Driver Operator
+include::modules/common-attributes.adoc[]
+:context: persistent-storage-csi-azure
+
+toc::[]
+
+== Overview
+
+{product-title} is capable of provisioning persistent volumes (PVs) using the Container Storage Interface (CSI) driver for Microsoft Azure Disk.
```

Comment:
In this first sentence only, I suggest writing out "Microsoft Azure Disk Storage" to match their latest terminology per: https://azure.microsoft.com/en-us/services/storage/disks/
For the rest of the doc, I think just "Azure Disk" more closely matches our CSI driver terminology.